src/tower_struggle/scenes/level_01.clj

```diff
@@ -20,11 +20,49 @@
     "img/big-background.png")
    (t/tetromino [300 800])])
 
+(defn tower-outline
+  [x y w h]
+  (let [tower-width w
+        tower-height h
+        coords {:tl  [x y]
+                :tr  [(+ x tower-width) y]
+                :bl  [x (+ y tower-height)]
+                :br  [(+ x tower-width) (+ y tower-height)]}
+        sides {:left   (concat (coords :tl) (coords :bl))
+               :right  (concat (coords :tr) (coords :br))
+               :top    (concat (coords :tl) (coords :tr))
+               :bottom (concat (coords :br) (coords :bl))}]
+    sides))
+
+(defn draw-tower-outline
+  "Draws a rectangle of dashed lines using Quil line primitives."
+  [x y w h]
+  (q/stroke common/black)
+  (let [o (tower-outline x y w h)
+        dash-length 10]
+    (doseq [[key value] o]
+      (let [distance (apply q/dist value)
```

Comment:
Really good choice to use the Quil `dist` function here, these are always tempting to write yourself but that way lies occasional bugs and common inefficiencies. This also shows you're reading the docs and getting to grips with the libs :+1: :medal_sports: